llumnix/backends/vllm/scheduler.py

```diff
@@ -21,8 +21,9 @@
 
 from llumnix.instance_info import InstanceInfo
 from llumnix.logger import init_logger
-from llumnix.llumlet.request import LlumnixRequest, RequestInferenceType
```

Comment:
so LlumnixRequest is only used in llumlet layer now?

Reply:
yes